simulator/util/dummy_perception_publisher_msgs/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+cmake_minimum_required(VERSION 3.5)
+project(dummy_perception_publisher_msgs)
+
+### Compile options
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wno-unused-parameter -Wall -Wextra -Wpedantic)
+endif()
+
+find_package(ament_cmake REQUIRED)
+find_package(rosidl_default_generators REQUIRED)
+find_package(autoware_perception_msgs REQUIRED)
+find_package(geometry_msgs REQUIRED)
+find_package(std_msgs REQUIRED)
+find_package(unique_identifier_msgs REQUIRED)
+
+rosidl_generate_interfaces(${PROJECT_NAME}
+  "msg/InitialState.msg"
+  "msg/Object.msg"
+  DEPENDENCIES std_msgs geometry_msgs autoware_perception_msgs unique_identifier_msgs
```

Comment:
Add in next line

      ADD_LINTER_TESTS

Reply:
This results in spurious errors. Omitted here and also removed in dependency autoware_perception_msgs.